notebooks/tuning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>I think this section looks nice.  Could be even more powerful with a short text section motivating the example however? eg - why are we stringing these transformers and forecasters together - is there a particular data analysis problem we are trying to solve? Would make it even more compelling to have a real example to connect the models to. (Doesn't have to be completely fleshed out, but a few sentences describing the data/why we might want to transform it this way could be nice).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>